tests/test_restore_names.py

```diff
@@ -1,3 +1,57 @@
+import copy
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_return_none():
+    assert restore_names([{
+        "first_name": "Jack",
+        "last_name": "Holy",
+        "full_name": "Jack Holy",
+    }]) is None
+
+
+def test_without_restore():
+    values = [
+        {
+            "first_name": "Jack",
+            "last_name": "Holy",
+            "full_name": "Jack Holy",
+        },
+        {
+            "first_name": "Mike",
+            "last_name": "Adams",
+            "full_name": "Mike Adams",
+        },
+    ]
+    values_for_test = copy.deepcopy(values)
```

Comment:
You don't need to use here `deepcopy`, just assign `expected` variable with expected result.